conformance/Makefile

```diff
@@ -76,8 +76,11 @@ run-conformance-tests: ## Run conformance tests
 	kubectl run -i conformance \
 		--image=$(PREFIX):$(TAG) --image-pull-policy=Never \
 		--overrides='{ "spec": { "serviceAccountName": "conformance" }  }' \
-		--restart=Never -- go test -v . -tags conformance -args --gateway-class=$(GATEWAY_CLASS) --debug \
-						        --supported-features=$(SUPPORTED_FEATURES)
+		--restart=Never -- sh -c "go test -v . -tags conformance,experimental -args --gateway-class=$(GATEWAY_CLASS) \
+						        --supported-features=$(SUPPORTED_FEATURES) --version=$(VERSION) \
```

Comment:
Where is VERSION defined?

Reply:
This is set by the pipeline when generating the report. Otherwise, local runs will just be empty string unless explicitly set.